testing/adios2/transform/TestBZip2Wrapper.cpp

```diff
@@ -0,0 +1,125 @@
+#include <cstdint>
+#include <iostream>
+#include <numeric> //std::iota
+#include <stdexcept>
+
+#include <adios2.h>
+
+#include <gtest/gtest.h>
+
```

Comment:
Correct!  Although it doesn't need to be a separate test, just 2 other test cases for the current test.  Much in the same way the engine tests have test cases for adios2 -> adios2, native -> adios2, adios2 -> native.  Either way it doesn't need to be a blocker for this branch, just a to-do after it's merged.  If you don't want to deal with it in this branch that's fine, I can just add an issue for the missing tests after the merge.